### PR TITLE
Tags has been changed following the specification

### PR DESCRIPTION
**Pull request by ypriverol**
@osallou Here I have changed all the recipes. Please review them and accept the PR.

### CODE REVIEW

File: comet/2016012/Dockerfile [hunk 1]
```diff
@@ -9,7 +9,7 @@ LABEL software.version="2016012"
 LABEL about.summary="an open source tandem mass spectrometry sequence database search tool"
 LABEL about.home="http://comet-ms.sourceforge.net/"
 LABEL about.documentation="http://comet-ms.sourceforge.net/parameters/parameters_2016010/"
-LABEL license="http://comet-ms.sourceforge.net/"
+LABEL about.license_file="http://comet-ms.sourceforge.net/"
 LABEL BIOTOOLS="https://bio.tools/comet"
```
Comment (osallou) [inline comet/2016012/Dockerfile]: should be extra.identifiers.biotools

File: test-ci/0.0.1/Dockerfile [hunk 2]
```diff
@@ -9,7 +9,7 @@ LABEL software.version="0.0.1"
 LABEL about.summary="A test container for internal testing"
 
 LABEL about.home="http://193.62.52.74/jenkins/"
-LABEL license="gpl3+"
+LABEL about.license_file="gpl3+"
```
Comment (osallou) [inline test-ci/0.0.1/Dockerfile:12]: this is about.license, not about.license_file

**Review by osallou — CHANGES_REQUESTED** [review]
most makes use of about.license_file, but do not specify about.license even if using standard licenses (gpl3 , etc.).
using license_file is good, but license should also be directly specified

**Comment by ypriverol** [comment]
I will change them, the problem is that I have noticed that most of them have some kind of modification in the license, when you read the license file. I will try to do my best to annotate them.

**Comment by osallou** [comment]
for custom ones, license should specify something like 'custom, see
license_file"

**Comment by ypriverol** [comment]
@osallou we are now ready to merge because I have updated all the license information. Please have a look and let me know.

File: biocontainers/Dockerfile [hunk 3]
```diff
@@ -9,7 +9,8 @@ LABEL software.version="08252016"
 LABEL about.summary="Base image for BioDocker"
```
Comment (osallou) [inline biocontainers/Dockerfile:9]: as a base image, shouldn't it follow the same rules as other containers ie having:
biocontainers/x.y.z/Dockerfile

to keep track of older versions. Automatic builds won't be able to detect version from directory

File: Rstudio-Proteomics/Dockerfile [hunk 4]
```diff
@@ -12,7 +12,8 @@ LABEL software.version="1"
 LABEL about.summary="Rstudio-Proteomics is an adaptation from the original images provided by the Rocker-org. The container provided here contains a Rstudio server configurated with several packages dedicated to Proteomcis data analysis.Rstudio-Proteomics is an adaptation from the original images provided by the Rocker-org. The container provided here contains a Rstudio server configurated with several packages dedicated to Proteomcis data analysis."
```
Comment (osallou) [inline Rstudio-Proteomics/Dockerfile:12]: this one does not follow rule to get version directory (Rstudio-Proteomics/x.y.z/Dockerfile)

File: pymol-jupiter/Dockerfile [hunk 5]
```diff
@@ -9,7 +9,8 @@ LABEL software.version="1"
 LABEL about.summary="a portable way of interacting with PyMol via Jupyter notebook"
```
Comment (osallou) [inline pymol-jupiter/Dockerfile:9]: missing version directory too

File: zorro/Dockerfile [hunk 6]
```diff
@@ -10,8 +10,10 @@ LABEL software.version="1.0"
 LABEL about.summary="A probabilistic alignment masking program"
```
Comment (osallou) [inline zorro/Dockerfile:10]: missing version directory too

**Comment by ypriverol** [comment]
I will update them

**Review by osallou — APPROVED** [review]

**Pull request merged**